### ISSUE TITLE
Cisco UCS Manager Report Issue

### ISSUE THREAD

**Issue by paulmx86** [AsBuiltReport/AsBuiltReport.Cisco.UcsManager#6]
**Describe the bug**
The Cisco UCS report fails while collecting information about Service Profile Templates.

**To Reproduce**
Run report against Cisco UCS Manager.

**Screenshots**
![image](https://user-images.githubusercontent.com/50809310/57988280-497bb480-7a5a-11e9-90ed-6f72d7dcf8d4.png)

Sample execution, with names removed.
![image](https://user-images.githubusercontent.com/50809310/57988343-14239680-7a5b-11e9-9b91-2bec2401bc01.png)

**Additional context**
Tested against Cisco UCS Manger v4.0(2b) and 3.1.
Installed latest Cisco UCS Powershell powertool suite.

This entire project / script is an absolutely brilliant effort.  Thank you for saving us users hours of work.

**Comment by tpcarman** [comment]
Hi @paulmx86,

If you connect to UCS Manager manually using the UCS cmdlets, what is the output of `Get-UcsServiceProfile | Where-Object {$_.Type -ne 'instance'} | Sort-Object Name`?

**Comment by paulmx86** [comment]
Hi @tpcarman,

I simplified the output due to length, but let me know if you need the whole thing.
PS C:\scripts> Get-UcsServiceProfile | Where-Object {$_.Type -ne 'instance'} | Sort-Object Name | select Name, Type

Name                Type
----                ----
compute-nodes       updating-template
compute-nodes-m5    updating-template
hx-nodes            updating-template
hx-nodes-m5         updating-template
TEMP                updating-template
US1-SPT-B-M5-ESX-V1 updating-template

**Comment by tpcarman** [comment]
Hi @paulmx86,

I believe I have located the issue.

Locate the `Invoke-AsBuiltReport.Cisco.UcsManager.ps1` file and modify `line 963` to be as follows;

```powershell
Section -Style Heading4 -Name "$($ServiceProfileTemplate.Name)" {
```

After modifying and saving the file, reload the module as follows;

```powershelll
Remove-Module -Name AsBuiltReport.Cisco.UcsManager -Force
Import-Module -Name AsBuiltReport.Cisco.UcsManager
```

Please let me know if this resolves your issue.

**Comment by paulmx86** [comment]
Worked perfectly.  Thank you!
I had tried this after staring at that section, but forgot to actually re-import the module.

**Comment by tpcarman** [comment]
Reopening issue to provide reference until the next update is released.

**Comment by tpcarman** [comment]
Resolved in 9e064d4
